README.md

```diff
@@ -125,7 +150,19 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+
+  lamp = Lamp.create(:wishes_remaining 3)
+  genie = Genie.create(:name "Genie")
+
+  genie.lamp = lamp
+  genie.save
+
+  lamp.update(:wishes_remaining 1)
+
+  Genie.create(:name "Jafar", :lamp Lamp.create(:wishes_remaining 3))
+
+  genie.update(:lamp nil)
+
 ```
```

Comment:
Careful with your syntax! You can either flip the colon to the other side of the symbol (`wishes_remaining: 3`) or leave it where it is and use hash rocket notation (`:wishes_remaining => 3`).